lib/strings.nix

```diff
@@ -269,6 +269,37 @@ rec {
     f:
     list: concatStringsSep sep (lib.imap1 f list);
 
+  /**
+    Like [`concatMapStringsSep`](#function-library-lib.strings.concatMapStringsSep)
+    but takes an attribute set instead of a list.
+
+    # Type
+
+    ```
+    concatMapAttrsStringSep :: String -> (String -> Any -> String) -> AttrSet -> String
+    ```
+
+    # Examples
+
+    :::{.example}
+    ## `lib.strings.concatMapAttrsStringSep` usage example
+
+    ```nix
+    concatMapAttrsStringSep "\n" (name: value: "${name}: foo-${value}") { a = "0.1.0"; b = "0.2.0"; }
+    => "a: foo-0.1.0\nb: foo-0.2.0"
+    ```
+
+    :::
+  */
+  concatMapAttrsStringSep =
+    # Separator to add between items
```

Comment:
Docs here don't get rendered anymore, check out other functions in this file to see how inputs are declared in the docs string

Reply:
Addressed.